pootle/templates/editor/units/edit.html

```diff
@@ -212,6 +212,7 @@
                 currentLocaleDir: '{{ language.direction }}',
                 sourceString: {{ unit.source_f|to_js }},
                 targetNplurals: {{ target_nplurals }},
+                fileType: '{{ filetype }}',
```

Comment:
I would suggest to leave this out of this PR: no component is aware of such prop but it will be passed down. Alternatively we could filter out unknown props in `reactEditor`, but I believe that approach is out of the scope of this PR.